src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -60,13 +70,23 @@ def _crawl(self) -> list[ClusterInfo]:
         all_clusters = list(self._ws.clusters.list())
         return list(self._assess_clusters(all_clusters))
 
+    def _get_cluster_policy_definition(self, policy_id):
```

Comment:
You don't need these methods. You feel policy once per id and assign it a variable